test/unit_tests/definition/definition.test.ts

```diff
@@ -0,0 +1,24 @@
+import { expect, test } from '@jest/globals';
+import { readFileSync } from 'fs';
+import { Position } from 'vscode-languageserver';
+import { Elaborate } from '../../../lib/elaborate/elaborate';
+import { findDefinitions } from '../../../lib/languageFeatures/findDefinition';
+import { ProjectParser } from '../../../lib/project-parser';
+import { defaultSettingsGetter } from '../../../lib/settings';
+import { VhdlLinter } from '../../../lib/vhdl-linter';
+
+test(`Testing definitions`, async () => {
+  const projectParser = await ProjectParser.create([__dirname], '', defaultSettingsGetter);
+  const linter = new VhdlLinter('definition.vhd', readFileSync(__dirname + '/definition.vhd', { encoding: 'utf8' }),
+    projectParser, defaultSettingsGetter);
+  await Elaborate.elaborate(linter);
+  for (const character of [13, 14, 15]) {
```

Comment:
```suggestion
  for (const character of [13, 14, 15, 21, 22]) {
```
also test the definition for the end of the word